gomanifest/build.go

```diff
@@ -14,10 +14,10 @@ func main() {
 	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
 
 	// Validate required number of parameters.
-	if len(os.Args) != 3 {
+	if len(os.Args) != 3 && len(os.Args) != 4 {
```

Comment:
```suggestion
	!(2 < len(os.Args) < 5)
```

Reply:
This is not allowed in go, if we need to avoid two len(), I need to store it in a variable and use it, changing code now.